client/helpers/scenes.ts

```diff
@@ -174,7 +165,8 @@ export interface QueuedSceneItem {
   readonly res?: Failure | QueuedJob;
 }
 
-const PollPercentage = 10;
+const defaultPolling: Polling = { intervalMs: 200, maxAttempts: 4500 }; // 15 minute timeout for batch completions
```

Comment:
This max attempt seems rather large, even for a polling interval. Should we consider a backoff implementation instead?